test/cli/remote-create/table-columns-flag/args

```diff
@@ -0,0 +1 @@
+remote create table --bucket in.c-bucket --name table1 --primary-key "" --columns 'col1,col2,col3' --storage-api-host %%TEST_KBC_STORAGE_API_HOST%% --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%%
```

Comment:
It should be `'"col1","col2","col3"'` instead of `'col1,col2,col3'`.

Reply:
I have tested many cases and this one stayed there

Reply:
Ok, but `'"col1","col2","col3"'` covers more problems, and it is use-case from the client ticket.